src/app/components/main-view/get/get.component.scss

```diff
@@ -33,6 +33,13 @@
       display: none;
     }
   }
+
+  td.truncate {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    max-width: 1px;
```

Comment:
What's the usage of 1px td element? You won't be able to see the text at all.

Reply:
To be totally honest, I have no idea why this works, but it does. If I remove just this line, the column expands to fit all the text it contains. The column is wider than 1 pixel though, I'm not sure why this value is used. I'm not sure that we'll do much better without requiring column widths for the whole table.

This code came from here: https://stackoverflow.com/a/19342181/95695

Reply:
Interesting. I wonder why is that.
Anyway, approved.